src/main/java/com/hotels/beeju/core/BeejuCore.java

```diff
@@ -133,6 +153,22 @@ public String connectionURL() {
     return connectionURL;
   }
 
+  public Path tempDir() {
+    return tempDir;
+  }
+
+  /**
+   * Delete temporary files used during test
+   */
+  private void deleteTempDir() throws IOException {
+    FileUtils.deleteDirectory(tempDir.toFile());
+    assertFalse(tempDir.toFile().exists());
```

Comment:
We shouldn't have an assertion in a class that isn't a unit test. If this assertion isn't already in a test for this class it should be put there.